diff --git a/CMakeLists.txt b/CMakeLists.txt
index 431eda87..ff8d678f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,7 +49,7 @@ include(FindPackageMessage)
 include(GNUInstallDirs)
 
 set(CONFIG_DIR "${CMAKE_INSTALL_FULL_SYSCONFDIR}")
-set(AVRDUDE_FULL_VERSION ${CMAKE_PROJECT_VERSION})
+set(AVRDUDE_FULL_VERSION ARDUINO_VERSION_PLACEHOLDER)
 
 # =====================================
 # Get Git commit info
@@ -91,7 +91,7 @@ if(Git_FOUND)
 
     # If the commit is not tagged, include the date and commit hash in the full version string.
     if(NOT GIT_COMMIT_HASH STREQUAL GIT_TAG_HASH)
-        set(AVRDUDE_FULL_VERSION "${CMAKE_PROJECT_VERSION}-${GIT_COMMIT_DATE} (${GIT_COMMIT_HASH})")
+        set(AVRDUDE_FULL_VERSION "ARDUINO_VERSION_PLACEHOLDER-${GIT_COMMIT_DATE} (${GIT_COMMIT_HASH})")
     endif()
 endif()
 
